boards/adi/max78002evkit/doc/index.rst

```diff
@@ -0,0 +1,297 @@
+
+  - Convolutional Neural Network (CNN) Accelerator
+
+    - Highly Optimized for Deep CNNs
+    - 2 Million 8-Bit Weight Capacity with 1-, 2-, 4-, and 8-bit Weights
+    - 1.3MB CNN Data Memory
+    - Programmable Input Image Size up to 2048 x 2048 Pixels
+    - Programmable Network Depth up to 128 Layers
+    - Programmable per Layer Network Channel Widths up to 1024 Channels
+    - 1- and 2-Dimensional Convolution Processing
+    - Capable of Processing VGA Images at 30fps
+
+  - Power Management for Extending Battery Life
+
+    - Integrated Single-Inductor Multiple-Output (SIMO) Switch-Mode Power Supply (SMPS)
+    - 2.85V to 3.6V Supply Voltage Range
+    - Support of Optional External Auxiliary CNN Power Supply
+    - Dynamic Voltage Scaling Minimizes Active Core Power Consumption
+    - 23.9μA/MHz While Loop Execution at 3.3V from Cache (CM4 only)
+    - Selectable SRAM Retention in Low-Power Modes with Real-Time Clock (RTC) Enabled
+
+  - Security and Integrity
+
+    - Available Secure Boot
+    - AES 128/192/256 Hardware Acceleration Engine
+    - True Random Number Generator (TRNG) Seed Generator
+
+  - Ultra-Low-Power Wireless Microcontroller
+
+    - Internal 100MHz Oscillator
+    - Flexible Low-Power Modes with 7.3728MHz System Clock Option
+    - 512KB Flash and 128KB SRAM (Optional ECC on One 32KB SRAM Bank)
+    - 16KB Instruction Cache
+
+  - Bluetooth 5.2 LE Radio
+
+    - Dedicated, Ultra-Low-Power, 32-Bit RISC-V Coprocessor to Offload Timing-Critical Bluetooth Processing
+    - Fully Open-Source Bluetooth 5.2 Stack Available
+    - Supports AoA, AoD, LE Audio, and Mesh
+    - High-Throughput (2Mbps) Mode
+    - Long-Range (125kbps and 500kbps) Modes
+    - Rx Sensitivity: -97.5dBm; Tx Power: +4.5dBm
+    - Single-Ended Antenna Connection (50Ω)
+
+  - Power Management Maximizes Battery Life
+
+    - 2.0V to 3.6V Supply Voltage Range
+    - Integrated SIMO Power Regulator
+    - Dynamic Voltage Scaling (DVS)
+    - 23.8μA/MHz Active Current at 3.0V
+    - 4.4μA at 3.0V Retention Current for 32KB
+    - Selectable SRAM Retention + RTC in Low-Power Modes
+
+  - Multiple Peripherals for System Control
+
+    - Up to Two High-Speed SPI Master/Slave
+    - Up to Three High-Speed I2C Master/Slave (3.4Mbps)
+    - Up to Four UART, One I2S Master/Slave
+    - Up to 8-Input, 10-Bit Sigma-Delta ADC 7.8ksps
+    - Up to Four Micro-Power Comparators
+    - Timers: Up to Two Four 32-Bit, Two LP, TwoWatchdog Timers
+    - 1-Wire® Master
+    - Up to Four Pulse Train (PWM) Engines
+    - RTC with Wake-Up Timer
+    - Up to 52 GPIOs
+
+  - Security and Integrity​
+
+    - Available Secure Boot
+    - TRNG Seed Generator
+    - AES 128/192/256 Hardware Acceleration Engine
+
+- External devices connected to the MAX78002 EVKIT:
+
+  - Color TFT Display
+  - Audio Stereo Codec Interface
+  - Digital Microphone
+  - A 8Mb QSPI ram
+
+Supported Features
+==================
+
+Below are the interfaces supported by Zephyr on MAX78002EVKIT.
```

Comment:
needs board name as used with west build, surrounding with \`\` and called board target

Reply:
Updated file, thanks.